Jenkinsfile

```diff
@@ -1181,20 +1184,25 @@ pipeline {
                         provisionNodes NODELIST: env.NODELIST,
                                        node_count: 1,
                                        snapshot: true,
-                                       inst_repos: daos_repos + ' ' + ior_repos,
+                                       inst_repos: el7_daos_repos + ' ' + ior_repos,
                                        inst_rpms: "ior-hpc mpich-autoload"
                         // Then just reboot the physical nodes
                         provisionNodes NODELIST: env.NODELIST,
                                        node_count: 9,
                                        power_only: true,
-                                       inst_repos: daos_repos + ' ' + ior_repos,
+                                       inst_repos: el7_daos_repos + ' ' + ior_repos,
                                        inst_rpms: "ior-hpc mpich-autoload"
                         runTest stashes: [ 'CentOS-install', 'CentOS-build-vars' ],
                                 script: '''test_tag=$(git show -s --format=%B | sed -ne "/^Test-tag-hw:/s/^.*: *//p")
                                            if [ -z "$test_tag" ]; then
                                                test_tag=pr,hw
                                            fi
                                            tnodes=$(echo $NODELIST | cut -d ',' -f 1-9)
+                                           sed -i -e 's/ofi+sockets/ofi+psm2/'                                                       \
+                                                  -e 's/\\(fabric_iface: *\\)[^ ]*\\( *.*\\)/\\1ib0\\2/'                             \
+                                                  -e '/FI_SOCKETS_CONN_TIMEOUT=/a\\ \\ - FI_PSM2_DISCONNECT=1\\n  - PSM2_MULTI_EP=1' \
+                                                     src/tests/ftest/data/daos_server_baseline.yaml
```

Comment:
Brian,  I believe that the FI_PSM2_DISCONNECT=1 is to be set on the client side only.   I don't think this should be in the server yaml.